### PR TITLE
Refactor aws modules

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: modules/aws/core/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Core
+
+This module is used to configure a standard public/private VPC and accompanying Route53 zome.
```
Comment (oscr) [inline modules/aws/core/README.md:3]: *zone

File: modules/aws/core/variables.tf [hunk 2]
```diff
@@ -4,22 +4,47 @@ variable "environment" {
...
-variable "dns_zone" {
-  description = "The DNS Zone that will be used by the EKS cluster"
-  type        = string
+variable "vpc_peering_enabled" {
+  description = "If true vpc peering will be configured"
+  type        = bool
+  default     = true
+}
+
+variable "vpc_peering_config" {
```
Comment (oscr) [inline modules/aws/core/variables.tf:38]: What if you need multiple VPC peerings?

File: modules/aws/core/README.md (hunk 1 above)
Comment (NissesSenap) [reply modules/aws/core/README.md:3]: thanks

**Review by NissesSenap — APPROVED** [review]
This is the first version, we will continue the development. Not ready for production

**Pull request merged**